### PR TITLE
Add conversational AI use case

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: _data/usecases.yml [hunk 1]
```diff
@@ -20,16 +20,17 @@
...
-    virtual assistants, and to answer some search queries. Modern question answerers
-    combine evaluation of transformer based language models with efficient retrieval of candidate text
-    snippets. Vespa makes it easy to build question answering systems combining
-    the state of the art in research with performance suitable for production.
+  link: "usecases#conversational-ai"
+  description: Large language models are revolutionary, but their usefulness for real tasks
+    is limited by their lack of memory, trustworthiness, specific knowledge, and ability
+    to reason in many steps. To make more useful agents we must give the models the ability
+    to store and search vector and text data in real time, and orchestrate many such
+    operations to carry out a task. Vespa is the ideal platform for this as it integrates
```
Comment (kkraune) [inline _data/usecases.yml:32]:
```suggestion
    operations to carry out a task. Vespa is the ideal platform for this, as it integrates 
```

File: usecases.html [hunk 2]
```diff
@@ -177,36 +177,26 @@ <h3>Recommendation sample applications</h3>
...
     <div>
       <div class="flex-column">
-        <h2 id="question-answering">Question answering</h2>
+        <h2 id="conversational-ai">Conversational AI</h2>
         <p>
-          Question answering provides direct answers to user's question. This is needed in chat-bots,
-          virtual assistants and similar, and is also becoming an expected feature of high-end search solutions, where
-          a direct answer is provided on queries that seem to be questions.
+          Large language models compress the information in large amounts of text into a handful of billions of parameters.
+          When done well this give them a certain amount of intelligence and true understanding, but they are still
```
Comment (kkraune) [inline usecases.html:183]:
```suggestion
          When done well, this gives them a certain amount of intelligence and true understanding, but they are still
```

File: usecases.html [hunk 3]
```diff
@@ -177,36 +177,26 @@ <h3>Recommendation sample applications</h3>
...
+          missing crucial ingredients to be truly useful in many settings: They need to be able to access specific
+          information about the topic at hand, form memories and temporary store information as they work,
```
Comment (kkraune) [inline usecases.html:185]:
```suggestion
          information about the topic at hand, form memories, and temporarily store information as they work,
```

File: usecases.html [hunk 4]
```diff
@@ -289,7 +279,42 @@ <h2 id="typeahead-suggestions">Typeahead suggestions</h2>
...
+          Question answering provides direct answers to user's question. This is needed in chat-bots,
+          virtual assistants and similar, and is also becoming an expected feature of high-end search solutions, where
+          a direct answer is provided on queries that seem to be questions.
+        </p>
+        <p>
+          A high quality question answerer works as follows: Text snippets are represented by vector embeddings
+          which are indexed for fast matching with
+          <a href="https://docs.vespa.ai/en/approximate-nn-hnsw.html">ANN</a>.
+          The best candidates found by ANN matching are evaluated in a
+          transformer based language model which outputs the score of the snippet as well as the beginning and
```
Comment (kkraune) [inline usecases.html:298]:
```suggestion
          transformer-based language model, which outputs the score of the snippet as well as the beginning and
```

File: usecases.html [hunk 5]
```diff
@@ -289,7 +279,42 @@ <h2 id="typeahead-suggestions">Typeahead suggestions</h2>
...
+        <p>
+          By using Vespa, the entire process can be implemented as an application on a
+          single platform, and made to execute with a latency of a few tens of milliseconds while scaling to any
+          volume, while delivering quality on par with the research state of the art.
+        </p>
+        <p>
+          See our
+          <a href="https://blog.vespa.ai/efficient-open-domain-question-answering-on-vespa/">blog post</a>
+          showing how to replicate the best question answering performance from the research community
+          as a production ready Vespa application, and
```
Comment (kkraune) [inline usecases.html:310]:
```suggestion
          showing how to replicate the best question-answering performance from the research community
          as a production-ready Vespa application, and
```

File: usecases.html [hunk 6]
```diff
@@ -289,7 +279,42 @@ <h2 id="typeahead-suggestions">Typeahead suggestions</h2>
...
+          <a href="https://blog.vespa.ai/from-research-to-production-scaling-a-state-of-the-art-machine-learning-system/">the followup post</a>
```
Comment (kkraune) [inline usecases.html:311]:
```suggestion
          <a href="https://blog.vespa.ai/from-research-to-production-scaling-a-state-of-the-art-machine-learning-system/">the follow-up post</a>
```

**Review by kkraune — APPROVED** [review]
These are great changes! I added some language nits from the robot ...

**Review by jobergum — APPROVED** [review]

**Comment by jobergum** [comment]
💯

**Pull request merged**